hack/e2e.go

```diff
@@ -156,7 +156,9 @@ func (t tester) getKubetest(get bool, old time.Duration) (string, error) {
 		return "", fmt.Errorf("cannot install kubetest until $GOPATH is set")
 	}
 	log.Print("Updating kubetest binary...")
-	cmd := []string{"go", "get", "-u", "k8s.io/test-infra/kubetest"}
+	// per: https://github.com/kubernetes/test-infra/pull/14094#issuecomment-526717609
+	// it is recommended to download kubetest with 1.11 modules and no -u
+	cmd := []string{"GO111MODULE=on", "go", "get", "k8s.io/test-infra/kubetest"}
```

Comment:
the GO111MODULE=on needs to be an environment variable, the command is still the same

Reply:
fixed